### ISSUE TITLE
Buttons consistency

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#287]
Define everything in detail and then assign.

**Comment by ForumG-SC** [comment]
@Rajat-Sen This ticket lacks the detailed information. As per my understanding the button's consistency was applied with this ticket (https://github.com/StatCan/StatcanPublicationUAT/issues/227) .

**Comment by Rajat-Sen** [comment]
Will add more details!

**Comment by Rajat-Sen** [comment]
@payeisa, What's t-shirt size?

**Comment by zhuelle** [comment]
@Rajat-Sen t-shirt size means work effort. Currently, the dev team is providing work effort or t-shirt size to help Isabelle to assign priority to any ticket.

@payeisa would you like that UX to provide work effort or t-shirt size as well? For this ticket specific, you would like to know how much time that Rajat spends to add more details?

**Comment by payeisa** [comment]
@zhuelle Great question! So far, I believe only the Dev team has been estimating, right? We have not included design work in the t-shirt sizes. Is this something we should start doing, or should we keep the t-shirt sizes for Dev only?

**Comment by zhuelle** [comment]
@payeisa You are right that currently, only Dev team is providing estimates for helping prioritization. I discussed with Rajat, it is ok to provide t-shirt sizes for UX work if required. I propose that we can keep the estimating process on all development tickets. For UX and BA, the effort can be provided when needed. Please let me know if you are ok with this approach. Thanks!
